examples/amp-story/interactives.html

```diff
@@ -16,6 +16,7 @@
     </style>
     <script async src="https://cdn.ampproject.org/v0.js"></script>
     <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <script async custom-element="amp-story-dev-tools" src="https://cdn.ampproject.org/v0/amp-story-dev-tools-0.1.js"></script>
```

Comment:
Needed for demoing the extension, we can keep this here to be able to trigger the extension on this story (because we have to load it like this on demos)